### ISSUE TITLE
Enter key should dismiss Run-inline panel

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#1936]
### Short Summary

Title

**Comment by Naatan** [comment]
Run-inline panel?

**Comment by Naatan** [comment]
oh, the lightning thing. We really gotta find a better name for that.

**Comment by cgchoffman** [comment]
It's called run in line everywhere is it not?  I don't use the lighting button.  I use Ctrl + B.

**Comment by Naatan** [comment]
Yeah it is :) I even came up with that name. I'm just saying the name is horrible and I should have come up with something better, cause I myself cannot even connect the dots.

**Comment by cgchoffman** [comment]
Oh, i liked it haha.  It makes total sense to me.  I didn't think there was an issue.  You're running it "in line" with the open file.  If you can think of a better name i'll buy you a beer...but i have to agree it's better.

**Comment by Naatan** [comment]
It makes sense when you use it in context, out of context it is confusing though.
